README.md

```diff
@@ -17,20 +17,20 @@ the square brackets (for multiple-choice questions).
 
 Select all that apply:
 ```
-[] Strings
-[] Booleans
+[x] Strings
+[x] Booleans
 [] Undefined
```

Comment:
Undefined and Integers are both actually a JS data type while Arrays are considered a data structure. 

The Answer should be: 
```
[X] Strings
[X] Booleans
[X] Undefined
[] NaN
[X] Integers
[] Arrays
[X] Null
```